src/redis_keys.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+#
+# igcollect - Redis keys
+#
+# Copyright (c) 2017 InnoGames GmbH
+#
+
+from argparse import ArgumentParser
+from subprocess import Popen, PIPE
+from time import time
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='redis')
+    parser.add_argument('--command', default='llen')
+    parser.add_argument('--keys', default='*queue*')
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+
+    template = args.prefix + '.{}.{} {} ' + str(int(time()))
+    queues = []
+    output = Popen(('redis-cli', 'keys', args.keys), stdout=PIPE).stdout.read()
+    for queue in output.splitlines():
+        data = Popen(
+            ['redis-cli', args.command, queue],
+            stdout=PIPE).stdout.read().strip('\n')
```

Comment:
This script would be more efficient by using the appropriate Python Redis driver and keeping connection open.